src/internal/tester/its/resourceService.go

```diff
@@ -0,0 +1,25 @@
+package its
+
+import "github.com/alcionai/corso/src/pkg/path"
+
+type ResourceServicer interface {
```

Comment:
Adding here since I can't comment directly on the `M365IntegTestSetup` struct definition... Given that we now have an interface type that allows getting the `IDs` and the service type (i.e. a useful but still minimal superset of data in `M365IntegTestSetup`) it seems like `M365IntegTestSetup` should actually be returning these for the different things it has instead of IDs. That may avoid the problem of having to setup these things separately in tests as they'd all be sourced from the `M365IntegTestSetup` struct

Reply:
Will make a note and follow-up on this later.